src/lib/Scenes/Artwork/Components/RequestConditionReport.tsx

```diff
@@ -0,0 +1,186 @@
+import { Button, Flex } from "@artsy/palette"
+import { RequestConditionReport_artwork } from "__generated__/RequestConditionReport_artwork.graphql"
+import { RequestConditionReport_me } from "__generated__/RequestConditionReport_me.graphql"
+import { RequestConditionReportMutation } from "__generated__/RequestConditionReportMutation.graphql"
+import { RequestConditionReportQuery } from "__generated__/RequestConditionReportQuery.graphql"
+import { Modal } from "lib/Components/Modal"
+import { defaultEnvironment } from "lib/relay/createEnvironment"
+import { Schema, track } from "lib/utils/track"
+import React from "react"
+import { View } from "react-native"
+import { commitMutation, createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import { PayloadError } from "relay-runtime"
+
+interface RequestConditionReportProps {
+  artwork: RequestConditionReport_artwork
+  me: RequestConditionReport_me
+}
+
+interface State {
+  requestingConditionReport: boolean
+  showConditionReportRequestedModal: boolean
+  showErrorModal: boolean
+  errorModalText: string
+}
+
+@track({
+  action_name: Schema.ActionNames.RequestConditionReport,
+  context_module: Schema.ContextModules.ArtworkDetails,
+})
+export class RequestConditionReport extends React.Component<RequestConditionReportProps, State> {
+  state = {
+    requestingConditionReport: false,
+    showConditionReportRequestedModal: false,
+    showErrorModal: false,
+    errorModalText: "",
+  }
+
+  requestConditionReport = () => {
+    const { artwork } = this.props
+    return new Promise(async (resolve, reject) => {
+      commitMutation<RequestConditionReportMutation>(defaultEnvironment, {
```

Comment:
As a matter of hygiene, this should use the relay environment passed in to this component as a prop. See the example in this file: https://github.com/artsy/emission/blob/813a5705c5469f3cf8c4917470c069e658600e84/src/lib/Components/Artist/ArtistHeader.tsx#L125

Otherwise one day someone could change the environment passed into the QueryRenderer and then this component would be using a separate environment for mutations which could cause confusing issues since environments have independent caches and network layers etc.